ctx.go

```diff
@@ -322,10 +366,10 @@ func (c ContextNoBody) QueryParamBoolErr(name string) (bool, error) {
 	return b, nil
 }
 
-func (c ContextNoBody) QueryParamBool(name string, defaultValue bool) bool {
+func (c ContextNoBody) QueryParamBool(name string) bool {
```

Comment:
Probbaly want to comment this similar to `QueryParamInt`. For instance what is the default value?

Reply:
The one defined in `option.QueryBool("myParam", "myDesc", param.Default(true))`. If not, `false`

It needs a comment yes